sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/functions/FunctionsTest.java

```diff
@@ -590,4 +640,389 @@ public void testSqrtDouble() {
                 """
         );
     }
+
+    @Test
+    public void testRound() {
+        this.qs("""
+                select round(15.1);
```

Comment:
are these functions documented?

Reply:
Yes. 
https://github.com/feldera/feldera/blob/e9beb9e63e12f95fd6a1308696e4e1b32fbbcaa6/docs/sql/decimal.md?plain=1#L86

Although the variant for `truncate` without the second parameter is missing in our docs. Will add that.